javascript/javascript1/week4/main.js

```diff
@@ -0,0 +1,137 @@
+let name;
+const todo = [];
+
+function getReply(command) {
+  // Conditions for 'Hello my name is ' command
+  if (command.includes("Hello my name is ")) {
+    if (name !== command.slice(17)) {
+      name = command.slice(17);
```

Comment:
We can reuse our code like this:

let localName=command.slice(17);
 if (name !== localName ) {
      name =localName;